src/progress-steps/styled-components.js

```diff
@@ -20,6 +20,8 @@ export const StyledProgressSteps = styled<{}>('ol', ({$theme}) => {
     paddingRight: $theme.sizing.scale500,
     paddingLeft: $theme.sizing.scale500,
     paddingBottom: $theme.sizing.scale300,
+    textAlign:
```

Comment:
do we need this line? shouldn't it be handled by the browser automatically?

Reply:
It isn't.